app/sass/style.scss

```diff
@@ -1,55 +1,59 @@
 body {
   font-family: 'Cabin', sans-serif;
-  
-  }
-
-nav {
-  text-align: center;
-  font-size: 16px;  
-  line-height: 16px;   
-  padding: 30px 0 0 0;
-  text-transform: uppercase;
-  
+  margin: 0;
+  color: #2E2E2E;
 }
 
-nav a {
-  padding: 10px 20px 10px 20px;
-  text-decoration: none;
-  color: black;
-  font-weight: 300;
+body::after {
+  content: "";
+  background: url("//lh6.googleusercontent.com/-iOeNF0I1KjY/VScOrd_2-JI/AAAAAAAAPrU/2e4rLM20I-s/w925-h617-no/brunchtable.jpeg") no-repeat center center fixed;
```

Comment:
Could not use the image stored in images due to errors so I stored externally and used the link for it, now image shows  up.